docs/reference-implementation/Developer-Guide.md

```diff
@@ -0,0 +1,58 @@
+# Developer Guide
+
+## Tools Required
+
+The following tools and applications must be installed on the client machine used to develop or deploy the AlwaysOn reference implementation:
+
+- Install [Azure CLI](https://docs.microsoft.com/cli/azure/service-page/azure%20cli?view=azure-cli-latest)
+
+- Install [Azure DevOps CLI](https://docs.microsoft.com/azure/devops/cli/?view=azure-devops)
+
+- Install [PowerShell](https://docs.microsoft.com/powershell/scripting/install/installing-powershell-on-windows?view=powershell-7.1).
+
+- Install [Visual Studio Code](https://code.visualstudio.com/Download)
+
+- Install [Terraform CLI](https://www.terraform.io/downloads)
+
+## Structure of the AlwaysOn reference implementation
```

Comment:
```suggestion
## Structure of the Azure Mission-Critical reference implementation
```
apart from the code artifacts themselves, we should only refer to it as Azure Mission-Critical now